test/thread/src/fiber_guard_page.d

```diff
@@ -8,25 +9,23 @@ else struct optStrategy { string a; }
 // (taken from core.thread)
 version = StackGrowsDown;
 
-enum stackSize = 4096;
-
 // Simple method that causes a stack overflow
 @optStrategy("none")
 void stackMethod()
 {
     // Over the stack size, so it overflows the stack
-    int[stackSize/int.sizeof+100] x;
+    int[minimumPageSize/int.sizeof+100] x;
```

Comment:
I think this might regress a PPC Bugfix I did yesterday. Can you test with a pageSize of 64kb?

Reply:
Can `alloca` with the runtime value `pageSize` be used instead?